pkg/cmd/admin/admin.go

```diff
@@ -59,6 +60,8 @@ func NewCommandAdmin(name, fullName string, out io.Writer) *cobra.Command {
 	cmds.AddCommand(admin.NewCommandCreateServerCert(admin.CreateServerCertCommandName, fullName+" "+admin.CreateServerCertCommandName, out))
 	cmds.AddCommand(admin.NewCommandCreateSignerCert(admin.CreateSignerCertCommandName, fullName+" "+admin.CreateSignerCertCommandName, out))
 
+	cmds.AddCommand(node.NewCommandManageNode(f, fullName, node.ManageNodeCommandName, out))
```

Comment:
`fullName` passed to `NewCommandManageNode` should be `fullName + " manage-node"`